core/modules/savers/put.js

```diff
@@ -58,21 +58,41 @@ PutSaver.prototype.save = function(text, method, callback) {
 		return false;
 	}
 	var self = this;
+	var uri = this.uri();
 	var headers = { "Content-Type": "text/html;charset=UTF-8" };
 	if(this.etag) {
 		headers["If-Match"] = this.etag;
 	}
 	$tw.utils.httpRequest({
-		url: this.uri(),
+		url: uri,
 		type: "PUT",
 		headers: headers,
 		data: text,
 		callback: function(err, data, xhr) {
 			if(err) {
 				callback(err);
 			} if(xhr.status === 200 || xhr.status === 201) {
```

Comment:
I did just test it with IIS webdav setting described here: https://www.youtube.com/playlist?list=PLuiC_HFhI4OwoVDb-B-VK0ydj-mBPNn-1 

There serve saves the file and returns 204 .... Which is different to the check above, but will invalidate the PR